src/style.css

```diff
@@ -1 +1,77 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #ffff;
```

Comment:
Shorthand hex notation abbreviates 6-character RRGGBB CSS colors into 3-character RGB shorthand, not 4